TECHNICAL-STEERING-COMMITTEE.md

```diff
@@ -1,36 +1,38 @@
 **Governance Overview**
  
-The Open Data Kit (ODK) project is governed by multiple governing committees. There are two categories of governing bodies, the Project Management Committee (PMC) and the Technical Steering Committees (TSC). The PMC is the ultimate authority over the project. The TSC is the authority over the technical direction of the project’s tool suite. 
+The Open Data Kit (ODK) project is governed by the Project Management Committee (PMC) and two Technical Steering Committees (ODK 1 TSC, ODK 2 TSC). The PMC is the ultimate authority over the project while the TSCs have authority over the technical direction of their respective suites of software: ODK 1 and ODK 2.
  
-_Note: As ODK transitions out of the University of Washington, the current PMC’s authority will be transferred to a Transition Board. Once the transition is finished, the authority will transfer to a more permanent governance body (likely another PMC). The TSC will remain active throughout the transition process and post-transition._
+_Note: As ODK transitions out of the University of Washington, the current PMC's authority will be transferred to a Transition Board. Once the transition is finished, the authority will transfer to a more permanent governance body (likely another PMC). The TSCs will remain active throughout the transition process and post-transition._
  
-**Technical Steering Committee**
+**Technical Steering Committees**
  
-The Technical Steering Committee (TSC) is responsible for high-level technical direction of their  tool suite. The TSC has authority over all technical aspects including:
-* Suite roadmap (feature addition/removal, tool addition/removal,incorporating community feedback,etc.).
-* Forming appropriate Working Groups (e.g., User Feedback, Documentation, Translation) to gather the necessary community feedback before making decisions.
-* Technical resources (e.g., code repositories, servers)
-* Maintaining the list of Committers
- 
-Wherever possible, the TSCs should choose shared policies and infrastructure which allow for consistent process, but administrative separation across tool suites. In the case where the TSCs cannot reach consensus on policies and infrastructure, the final decision will be made by a majority vote by the PMC.
- 
-Although the TSC may update the TSC governance (e.g. this document) as it finds appropriate, revisions are subject to PMC review and veto, as ultimate authority over governing the ODK project rests with the PMC. Any update to the TSC governing document must be approved by the majority of the PMC. It is expected that the TSC will have a collaborative relationship (perhaps some overlap in membership) with the PMC.
```

Comment:
I'm going to push back here because I think adding it back makes the document harder to read. Or rather, when you read it with the old text, it's redundant since we've made it explict in a lot of places already.

At the very top, it says. 
>  The PMC is the ultimate authority over the project

Then in the relevant section, it says,
>  In the case where the TSCs cannot reach consensus...the final decision will be made by a majority vote by the PMC.

Then it goes on to say...
> Any update to the TSC governing document must be approved by the majority of the PMC.

I can't imagine the scenario where there is a dispute between the TSC where it isn't clear that the PMC is the decider. Is there a scenario that you have in mind?

Reply:
Fair points which is why I made it only a comment instead of a need changes, I was looking for discussion on it. 

My comment was this was a concern when we went through the draft/revision the first time and I wanted to make sure it got enough consideration this time. 

I am fine with it as you do point out it is covered in other areas.